website/docs/docs/dbt-cloud-apis/sl-jdbc.md

```diff
@@ -147,9 +147,44 @@ select * from {{
 }}
 ```
 
-</DetailsToggle>
+</Expandable>
+
+<Expandable alt_header="Fetch metrics by substring search">
+
+You can filter your metrics to include only those that contain a specific substring. Use the `search` argument to specify the substring you want to match.
```

Comment:
```suggestion
You can filter your metrics to include only those that contain a specific [substring](/reference/dbt-jinja-functions/cross-database-macros#position). Use the `search` argument to specify the substring you want to match.
```

Reply:
Hiya @mirnawong1 

This is a non-blocking suggestion but I thought it might be a good idea to link out the word substring in case the user doesn't know what it is.

Kind Regards
Natalie